tests/notifications_python_client/test_base_api_client.py

```diff
@@ -9,8 +9,8 @@
 
 
 @pytest.mark.parametrize('client', [
-    BaseAPIClient(api_key=COMBINED_API_KEY),
-    BaseAPIClient(COMBINED_API_KEY),
+    BaseAPIClient(api_key=COMBINED_API_KEY, timeout=10),
+    BaseAPIClient(COMBINED_API_KEY, timeout=10),
```

Comment:
This is unrelated to the purpose of this test: checking we pass through `service_id` and `key`. We're also not checking `timeout` is adopted correctly, which you could do with `mocker` or just test the value is at least set on the client object instance.

How about checking this in its own test?

Reply:
Thank you so much for responding! I updated the tests to check that the default time out is 30/ when it is set it is the correct value on the instance.